### ISSUE TITLE
SOP : indicate more clearly additional information that can be provided outside the yaml template

### ISSUE THREAD

**Issue by lheisler** [icgc-dcc/egasub#67]
EGA allows additional information to be provided when registering objects, outside of what is indicated in the yaml templates for analysis and run objects and explained in the SOP.

for example, I've added the following lines to my analysis.yaml, to provide some detail on how the analysis was done

analysis_attributes:
  analysis_attribute:
    tag: aligner
    value: bwa
  analysis_attribute:
    tag: aligner_ver
    value: 0.6.2
  analysis_attribute:
    tag: mark_duplicates
    value: picard
  analysis_attribute:
    tag: mark_duplicates_ver
    value: 1.90

This was info i had provided previously when using XML through the rest server

It would be good to indicate that this is possile in the SOP.

**Comment by junjun-zhang** [comment]
Yes, EGA metadata XML (SRA XML in general) allows submitter to add custom key-value pair. We plan to support that by adding  `sample.attributes`, `experiment.attributes` and `analysis.attributes` to the metadata YAML. It will be necessary to preserve some keys, eg, `icgc_sample_id` which we use to add ICGC Sample ID as custom key-value attribute for `sample` object.
